packages/validation-manager/src/altmempool/AltMempoolConfig.ts

```diff
@@ -25,15 +27,50 @@ export interface BaseAltMempoolRule {
 }
 
 export interface AltMempoolConfig {
-  [mempoolId: number]: { [rule in ERC7562Rule]?: BaseAltMempoolRule }
+  [mempoolId: string]: { [rule in ERC7562Rule]?: BaseAltMempoolRule }
```

Comment:
using erc7562 rules as index is not that great: there is no easy 1-1 mapping from rule to code: many rules have code split in several places, and implementation that relates to multiple rules.